autobahn/twisted/wamp.py

```diff
@@ -54,7 +54,7 @@
 except ImportError:
     # Not on PY3 yet
     service = None
-    __all__.pop(__all__.index("Service"))
+    __all__ = tuple(list(__all__).pop(__all__.index('Service')))
 
```

Comment:
A perhaps simpler option here is to just change `__all__` to be a list instead of a tuple; it can be either.